### PR TITLE
expose queue metrics via statsd every 60 seconds

### PR DESCRIPTION
**Pull request by thombergs**
Exposes some queue stats every 60 seconds.

### CODE REVIEW

**Review by KAbakumovAtl — DISMISSED** [review]
Looks good to me, well done. let's test it on Staging

File: src/worker/main.ts [hunk 1]
```diff
@@ -127,11 +128,51 @@ const sentryMiddleware = (jobHandler) => async (job) => {
...
+
+			await (Promise.all([
+				queue.getActiveCount().then((count: number) => counts.active = count),
+				queue.getDelayedCount().then((count: number) => counts.delayed = count),
+				queue.getFailedCount().then((count: number) => counts.failed = count),
+				queue.getCompletedCount().then((count: number) => counts.completed = count),
+				queue.getPausedCount().then((count: number) => counts.paused = count),
+				queue.getRepeatableCount().then((count: number) => counts.repeatable = count),
+				queue.getWaitingCount().then((count: number) => counts.waiting = count),
+			]))
```
Comment (thombergs) [inline src/worker/main.ts:153]: I would have liked to use Bull's function `getJobCountByTypes()` to get all counts at once, but it returns a single value 0 in my tests for whatever reason ...

**Comment by thombergs** [comment]
tested in `ddev`: 
image

File: src/worker/main.ts (hunk 1 above)
Comment (mboudreau) [reply src/worker/main.ts:154]: Couldn't you just do `getJobCounts()` instead?

**Review by KAbakumovAtl — APPROVED** [review]

**Review by bgvozdev — APPROVED** [review]

**Review by mboudreau — APPROVED** [review]

**Pull request merged**